adview-manager/src/lib.rs

```diff
@@ -234,7 +234,7 @@ pub fn get_unit_html_with_events(
             .map(|validator| {
                 let fetch_url = format!(
                     "{}/channel/{}/events?pubAddr={}",
```

Comment:
This will be fixed when the `events` route for a `Campaign` is updated, see tasks for #382